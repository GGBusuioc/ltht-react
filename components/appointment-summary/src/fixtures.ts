import { Encounter, EncounterStatusCode, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const AppointmentOne: Encounter = {
  id: '346ca927-67e0-e931-a2c7-115087226fg2|appointment',
  metadata: mockMetadata,
  type: [
    {
      text: 'See GP regarding sore throat',
      coding: [{ display: 'GP', code: '64706' }],
    },
  ],
  period: {
    start: {
      value: '2020-04-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  serviceProvider: {
    display: 'Beeston Village Surgery',
    typeName: 'GP',
  },
  status: EncounterStatusCode.Planned,
}

const AppointmentTwo: Encounter = {
  id: '941ca927-34e0-e911-a2c7-005087226fe4|appointment',
  metadata: mockMetadata,
  type: [
    {
      text: 'Chest x-ray for recent car accident',
      coding: [{ display: 'Hospital', code: '26706' }],
    },
  ],
  period: {
    start: {
      value: '2019-12-12T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  serviceProvider: {
    display: 'Chapel Allerton Hospital',
    typeName: 'Hospital',
  },
  status: EncounterStatusCode.Finished,
}

export { AppointmentOne, AppointmentTwo }
