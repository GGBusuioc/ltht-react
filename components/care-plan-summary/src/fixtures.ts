import { CarePlan, CarePlanStatusCode, PartialDateTimeKindCode, CarePlanIntentCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const CarePlanOne: CarePlan = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|careplan',
  metadata: mockMetadata,
  author: { display: 'Joe Bloggs', type: 'Test' },
  intent: CarePlanIntentCode.Plan,
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  status: CarePlanStatusCode.Active,
  title: 'Test Care Plan Summary One',
}

const CarePlanTwo: CarePlan = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|careplan',
  metadata: mockMetadata,
  author: { display: 'John Mclane', type: 'Test' },
  intent: CarePlanIntentCode.Proposal,
  period: {
    start: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2015-03-04T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  status: CarePlanStatusCode.Completed,
  title: 'Test Care Plan Summary Two',
}

export { CarePlanOne, CarePlanTwo }
