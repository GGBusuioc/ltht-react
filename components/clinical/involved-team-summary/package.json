{
  "name": "@ltht-react/involved-team-summary",
  "version": "0.1.26",
  "description": "ltht-react clinical InvolvedTeamSummary component.",
  "author": "LTHT",
  "homepage": "",
  "license": "MIT",
  "main": "lib/index.js",
  "typings": "lib/index.d.ts",
  "files": [
    "lib"
  ],
  "directories": {
    "lib": "lib"
  },
  "publishConfig": {
    "access": "public"
  },
  "scripts": {
    "clean": "rimraf lib && rimraf node_modules",
    "build": "tsc"
  },
  "dependencies": {
    "@emotion/core": "^10.0.27",
    "@emotion/styled": "^10.0.27",
    "@ltht-react/list": "^0.1.22",
    "@ltht-react/styles": "^0.1.16",
    "@ltht-react/summary": "^0.0.20",
    "@ltht-react/types": "^0.0.23",
    "@ltht-react/utils": "^0.1.16",
    "emotion": "^10.0.27",
    "react": "^16.12.0"
  },
  "gitHead": "d9d251dbbc5bf166a597a85f0af167e9e2f9824c"
}
