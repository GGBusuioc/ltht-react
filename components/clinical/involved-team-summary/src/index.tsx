import React from 'react'

import { EpisodeOfCare } from '@ltht-react/types'
import { ListItem } from '@ltht-react/list'
import { Card, CardHeader, CardList } from '@ltht-react/card'
import InvolvedTeamSummaryItem from './molecules/involved-team-summary-item'

const InvolvedTeamSummary: React.FC<Props> = ({ title = 'Involved Teams', episodeOfCares = [], clickHandler }) => {
  return (
    <Card noData={episodeOfCares.length === 0}>
      <CardHeader>
        <h3>{title}</h3>
      </CardHeader>
      <CardList clickable={!!clickHandler}>
        {episodeOfCares.map(episodeOfCare => (
          <ListItem key={episodeOfCare.id}>
            <InvolvedTeamSummaryItem episodeOfCare={episodeOfCare} clickHandler={clickHandler} />
          </ListItem>
        ))}
      </CardList>
    </Card>
  )
}

interface Props {
  title?: string
  episodeOfCares: EpisodeOfCare[] | undefined
  clickHandler?(episodeOfCare: EpisodeOfCare): void
}

export default InvolvedTeamSummary
