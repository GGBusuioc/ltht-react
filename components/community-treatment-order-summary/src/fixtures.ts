import {
  LypftCommunityTreatmentOrder,
  CarePlanStatusCode,
  PartialDateTimeKindCode,
  CarePlanIntentCode,
  Metadata,
} from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const CommunityTreatmentOrderOne: LypftCommunityTreatmentOrder = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|communityTreatmentOrder',
  metadata: mockMetadata,
  consentToTreat: 'Consent Not Given',
  legalStatus: {
    coding: [{ display: 'Approved', code: '3309007' }],
    text: 'Approved',
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  restrictions: 'No restrictions',
}

const CommunityTreatmentOrderTwo: LypftCommunityTreatmentOrder = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|communityTreatmentOrder',
  metadata: mockMetadata,
  consentToTreat: 'Consent Given',
  legalStatus: {
    coding: [{ display: 'Pending', code: '779007' }],
    text: 'Pending',
  },
  period: {
    start: {
      value: '2020-01-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  restrictions: 'No Restrictions',
}

export { CommunityTreatmentOrderOne, CommunityTreatmentOrderTwo }
