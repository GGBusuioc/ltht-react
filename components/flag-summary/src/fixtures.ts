import { Flag, FlagStatusCode, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const FlagOne: Flag = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard',
  metadata: mockMetadata,
  author: {
    display: 'John Smith',
    reference:
      'https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser',
    type: 'Practitioner',
    identifier: undefined,
  },
  code: {
    coding: [{ display: 'Lives in Known High Risk Area', code: '109006' }],
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Year,
    },
  },
  status: FlagStatusCode.Inactive,
}

const FlagTwo: Flag = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard',
  metadata: mockMetadata,
  author: {
    display: 'John Smith',
    reference:
      'https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser',
    type: 'Practitioner',
    identifier: undefined,
  },
  code: {
    coding: [{ display: 'High Risk of Violence', code: '56706' }],
  },
  category: {
    coding: [{ display: 'Violence', code: '66706' }],
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },

  status: FlagStatusCode.Inactive,
}

export { FlagOne, FlagTwo }
