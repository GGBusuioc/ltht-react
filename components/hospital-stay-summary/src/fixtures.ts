import { HospitalStay, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const HospitalStayOne: HospitalStay = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|hospitalstay',
  metadata: mockMetadata,
  nhsNumber: '123456789',
  serviceProvider: { display: 'Chapel Allerton Hospital', type: 'Hospital' },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
}

const HospitalStayTwo: HospitalStay = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|hospitalstay',
  metadata: mockMetadata,
  nhsNumber: '987654321',
  serviceProvider: { display: 'Beaumont Hospital', type: 'Hospital' },
  period: {
    start: {
      value: '2020-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
}

export { HospitalStayOne, HospitalStayTwo }
