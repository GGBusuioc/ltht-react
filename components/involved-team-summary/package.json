{
  "name": "@ltht-react/involved-team-summary",
  "version": "0.0.0",
  "description": "ltht-react InvolvedTeamSummary component.",
  "author": "LTHT",
  "homepage": "",
  "license": "MIT",
  "main": "lib/index.js",
  "typings": "lib/index.d.ts",
  "files": [
    "lib"
  ],
  "directories": {
    "lib": "lib"
  },
  "publishConfig": {
    "access": "public"
  },
  "scripts": {
    "clean": "rimraf lib",
    "build": "yarn build:lib",
    "watch": "tsc -w",
    "build:lib": "tsc",
    "type:check": "tsc --noEmit",
    "lint": "prettylint 'src/*.{ts,tsx}'"
  },
  "dependencies": {
    "@emotion/core": "^10.0.27",
    "@emotion/styled": "^10.0.27",
    "@ltht-react/styles": "^0.0.1",
    "@ltht-react/types": "^0.0.4",
    "@ltht-react/utils": "^0.0.4",
    "@ltht-react/widget": "^0.0.0",
    "emotion": "^10.0.27",
    "react": "^16.12.0"
  }
}