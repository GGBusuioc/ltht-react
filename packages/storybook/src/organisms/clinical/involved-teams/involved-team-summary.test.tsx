import React from 'react'
import { mount } from 'enzyme'
import ReactDOM from 'react-dom'

import InvolvedTeamSummary from '@ltht-react/involved-team-summary'
import { InvolvedTeamOne, InvolvedTeamTwo } from './involved-team-summary.fixtures'

describe('InvolvedTeamSummary', () => {
  const episodeOfCares = [InvolvedTeamOne, InvolvedTeamTwo]
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<InvolvedTeamSummary episodeOfCares={episodeOfCares} />, div)
  })
  it('matches snapshot', () => {
    expect(mount(<InvolvedTeamSummary episodeOfCares={episodeOfCares} />)).toMatchSnapshot('wrapper mount')
  })
})
