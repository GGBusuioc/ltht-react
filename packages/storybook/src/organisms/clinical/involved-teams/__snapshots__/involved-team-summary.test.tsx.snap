// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InvolvedTeamSummary matches snapshot: wrapper mount 1`] = `
.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.emotion-4 > div {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.emotion-0 {
  color: #231f20;
  text-align: left;
}

.emotion-1 {
  color: -v-a-l-u-e:#768692;
  -d-a-r-k-e-r25: #59656e;
  -l-i-g-h-t-e-r25: #98a4ad;
  text-align: left;
}

.emotion-2 {
  color: #231f20;
  text-align: right;
}

.emotion-3 {
  color: -v-a-l-u-e:#768692;
  -d-a-r-k-e-r25: #59656e;
  -l-i-g-h-t-e-r25: #98a4ad;
  text-align: right;
}

<InvolvedTeamSummary
  episodeOfCares={
    Array [
      Object {
        "careManager": Object {
          "display": "Bella Swann (0113 111111)",
          "typeName": "Care Manager",
        },
        "id": "580ca927-34e0-e911-a2c7-005056926fe4|involvedteam",
        "metadata": Object {
          "dataSources": Array [
            Object {
              "display": "Mock",
            },
          ],
          "isRedacted": true,
          "requestedWhen": "",
        },
        "period": Object {
          "end": Object {
            "kind": "DATE",
            "value": "2020-01-02T00:00:00+00:00",
          },
          "start": Object {
            "kind": "DATE",
            "value": "2018-12-01T00:00:00+00:00",
          },
        },
        "status": "FINISHED",
        "team": Array [
          Object {
            "display": "LYPFT Outbound Care Team",
            "typeName": "Test Team",
          },
        ],
        "type": Array [
          Object {
            "coding": Array [
              Object {
                "code": "109007",
                "display": "Community Care",
              },
            ],
          },
        ],
      },
      Object {
        "careManager": Object {
          "display": "Joe Bloggs (0113 000000)",
          "typeName": "Care Manager",
        },
        "id": "690ca927-68e0-c511-b2c7-898056926ze4|involvedteam",
        "metadata": Object {
          "dataSources": Array [
            Object {
              "display": "Mock",
            },
          ],
          "isRedacted": true,
          "requestedWhen": "",
        },
        "period": Object {
          "start": Object {
            "kind": "DATE",
            "value": "2019-12-01T00:00:00+00:00",
          },
        },
        "status": "ACTIVE",
        "team": Array [
          Object {
            "display": "LYPFT Inbound Care Team",
            "typeName": "Test Team",
          },
        ],
        "type": Array [
          Object {
            "coding": Array [
              Object {
                "code": "109016",
                "display": "Home Care",
              },
            ],
          },
        ],
      },
    ]
  }
>
  <ListItem
    key="580ca927-34e0-e911-a2c7-005056926fe4|involvedteam"
  >
    <li>
      <InvolvedTeamSummaryItem
        episodeOfCare={
          Object {
            "careManager": Object {
              "display": "Bella Swann (0113 111111)",
              "typeName": "Care Manager",
            },
            "id": "580ca927-34e0-e911-a2c7-005056926fe4|involvedteam",
            "metadata": Object {
              "dataSources": Array [
                Object {
                  "display": "Mock",
                },
              ],
              "isRedacted": true,
              "requestedWhen": "",
            },
            "period": Object {
              "end": Object {
                "kind": "DATE",
                "value": "2020-01-02T00:00:00+00:00",
              },
              "start": Object {
                "kind": "DATE",
                "value": "2018-12-01T00:00:00+00:00",
              },
            },
            "status": "FINISHED",
            "team": Array [
              Object {
                "display": "LYPFT Outbound Care Team",
                "typeName": "Test Team",
              },
            ],
            "type": Array [
              Object {
                "coding": Array [
                  Object {
                    "code": "109007",
                    "display": "Community Care",
                  },
                ],
              },
            ],
          }
        }
      >
        <div
          className="emotion-4"
          role="link"
        >
          <div>
            <InvolvedTeamTitle
              episodeOfCare={
                Object {
                  "careManager": Object {
                    "display": "Bella Swann (0113 111111)",
                    "typeName": "Care Manager",
                  },
                  "id": "580ca927-34e0-e911-a2c7-005056926fe4|involvedteam",
                  "metadata": Object {
                    "dataSources": Array [
                      Object {
                        "display": "Mock",
                      },
                    ],
                    "isRedacted": true,
                    "requestedWhen": "",
                  },
                  "period": Object {
                    "end": Object {
                      "kind": "DATE",
                      "value": "2020-01-02T00:00:00+00:00",
                    },
                    "start": Object {
                      "kind": "DATE",
                      "value": "2018-12-01T00:00:00+00:00",
                    },
                  },
                  "status": "FINISHED",
                  "team": Array [
                    Object {
                      "display": "LYPFT Outbound Care Team",
                      "typeName": "Test Team",
                    },
                  ],
                  "type": Array [
                    Object {
                      "coding": Array [
                        Object {
                          "code": "109007",
                          "display": "Community Care",
                        },
                      ],
                    },
                  ],
                }
              }
            >
              <div
                className="emotion-0"
              >
                LYPFT Outbound Care Team
              </div>
            </InvolvedTeamTitle>
            <InvolvedTeamDescription
              episodeOfCare={
                Object {
                  "careManager": Object {
                    "display": "Bella Swann (0113 111111)",
                    "typeName": "Care Manager",
                  },
                  "id": "580ca927-34e0-e911-a2c7-005056926fe4|involvedteam",
                  "metadata": Object {
                    "dataSources": Array [
                      Object {
                        "display": "Mock",
                      },
                    ],
                    "isRedacted": true,
                    "requestedWhen": "",
                  },
                  "period": Object {
                    "end": Object {
                      "kind": "DATE",
                      "value": "2020-01-02T00:00:00+00:00",
                    },
                    "start": Object {
                      "kind": "DATE",
                      "value": "2018-12-01T00:00:00+00:00",
                    },
                  },
                  "status": "FINISHED",
                  "team": Array [
                    Object {
                      "display": "LYPFT Outbound Care Team",
                      "typeName": "Test Team",
                    },
                  ],
                  "type": Array [
                    Object {
                      "coding": Array [
                        Object {
                          "code": "109007",
                          "display": "Community Care",
                        },
                      ],
                    },
                  ],
                }
              }
            >
              <div
                className="emotion-1"
              >
                Bella Swann (0113 111111)
              </div>
            </InvolvedTeamDescription>
          </div>
          <div>
            <InvolvedTeamPeriod
              episodeOfCare={
                Object {
                  "careManager": Object {
                    "display": "Bella Swann (0113 111111)",
                    "typeName": "Care Manager",
                  },
                  "id": "580ca927-34e0-e911-a2c7-005056926fe4|involvedteam",
                  "metadata": Object {
                    "dataSources": Array [
                      Object {
                        "display": "Mock",
                      },
                    ],
                    "isRedacted": true,
                    "requestedWhen": "",
                  },
                  "period": Object {
                    "end": Object {
                      "kind": "DATE",
                      "value": "2020-01-02T00:00:00+00:00",
                    },
                    "start": Object {
                      "kind": "DATE",
                      "value": "2018-12-01T00:00:00+00:00",
                    },
                  },
                  "status": "FINISHED",
                  "team": Array [
                    Object {
                      "display": "LYPFT Outbound Care Team",
                      "typeName": "Test Team",
                    },
                  ],
                  "type": Array [
                    Object {
                      "coding": Array [
                        Object {
                          "code": "109007",
                          "display": "Community Care",
                        },
                      ],
                    },
                  ],
                }
              }
            >
              <div
                className="emotion-2"
              >
                01-Dec-2018 to 02-Jan-2020
              </div>
            </InvolvedTeamPeriod>
            <InvolvedTeamType
              episodeOfCare={
                Object {
                  "careManager": Object {
                    "display": "Bella Swann (0113 111111)",
                    "typeName": "Care Manager",
                  },
                  "id": "580ca927-34e0-e911-a2c7-005056926fe4|involvedteam",
                  "metadata": Object {
                    "dataSources": Array [
                      Object {
                        "display": "Mock",
                      },
                    ],
                    "isRedacted": true,
                    "requestedWhen": "",
                  },
                  "period": Object {
                    "end": Object {
                      "kind": "DATE",
                      "value": "2020-01-02T00:00:00+00:00",
                    },
                    "start": Object {
                      "kind": "DATE",
                      "value": "2018-12-01T00:00:00+00:00",
                    },
                  },
                  "status": "FINISHED",
                  "team": Array [
                    Object {
                      "display": "LYPFT Outbound Care Team",
                      "typeName": "Test Team",
                    },
                  ],
                  "type": Array [
                    Object {
                      "coding": Array [
                        Object {
                          "code": "109007",
                          "display": "Community Care",
                        },
                      ],
                    },
                  ],
                }
              }
            >
              <div
                className="emotion-3"
              >
                Finished
              </div>
            </InvolvedTeamType>
          </div>
        </div>
      </InvolvedTeamSummaryItem>
    </li>
  </ListItem>
  <ListItem
    key="690ca927-68e0-c511-b2c7-898056926ze4|involvedteam"
  >
    <li>
      <InvolvedTeamSummaryItem
        episodeOfCare={
          Object {
            "careManager": Object {
              "display": "Joe Bloggs (0113 000000)",
              "typeName": "Care Manager",
            },
            "id": "690ca927-68e0-c511-b2c7-898056926ze4|involvedteam",
            "metadata": Object {
              "dataSources": Array [
                Object {
                  "display": "Mock",
                },
              ],
              "isRedacted": true,
              "requestedWhen": "",
            },
            "period": Object {
              "start": Object {
                "kind": "DATE",
                "value": "2019-12-01T00:00:00+00:00",
              },
            },
            "status": "ACTIVE",
            "team": Array [
              Object {
                "display": "LYPFT Inbound Care Team",
                "typeName": "Test Team",
              },
            ],
            "type": Array [
              Object {
                "coding": Array [
                  Object {
                    "code": "109016",
                    "display": "Home Care",
                  },
                ],
              },
            ],
          }
        }
      >
        <div
          className="emotion-4"
          role="link"
        >
          <div>
            <InvolvedTeamTitle
              episodeOfCare={
                Object {
                  "careManager": Object {
                    "display": "Joe Bloggs (0113 000000)",
                    "typeName": "Care Manager",
                  },
                  "id": "690ca927-68e0-c511-b2c7-898056926ze4|involvedteam",
                  "metadata": Object {
                    "dataSources": Array [
                      Object {
                        "display": "Mock",
                      },
                    ],
                    "isRedacted": true,
                    "requestedWhen": "",
                  },
                  "period": Object {
                    "start": Object {
                      "kind": "DATE",
                      "value": "2019-12-01T00:00:00+00:00",
                    },
                  },
                  "status": "ACTIVE",
                  "team": Array [
                    Object {
                      "display": "LYPFT Inbound Care Team",
                      "typeName": "Test Team",
                    },
                  ],
                  "type": Array [
                    Object {
                      "coding": Array [
                        Object {
                          "code": "109016",
                          "display": "Home Care",
                        },
                      ],
                    },
                  ],
                }
              }
            >
              <div
                className="emotion-0"
              >
                LYPFT Inbound Care Team
              </div>
            </InvolvedTeamTitle>
            <InvolvedTeamDescription
              episodeOfCare={
                Object {
                  "careManager": Object {
                    "display": "Joe Bloggs (0113 000000)",
                    "typeName": "Care Manager",
                  },
                  "id": "690ca927-68e0-c511-b2c7-898056926ze4|involvedteam",
                  "metadata": Object {
                    "dataSources": Array [
                      Object {
                        "display": "Mock",
                      },
                    ],
                    "isRedacted": true,
                    "requestedWhen": "",
                  },
                  "period": Object {
                    "start": Object {
                      "kind": "DATE",
                      "value": "2019-12-01T00:00:00+00:00",
                    },
                  },
                  "status": "ACTIVE",
                  "team": Array [
                    Object {
                      "display": "LYPFT Inbound Care Team",
                      "typeName": "Test Team",
                    },
                  ],
                  "type": Array [
                    Object {
                      "coding": Array [
                        Object {
                          "code": "109016",
                          "display": "Home Care",
                        },
                      ],
                    },
                  ],
                }
              }
            >
              <div
                className="emotion-1"
              >
                Joe Bloggs (0113 000000)
              </div>
            </InvolvedTeamDescription>
          </div>
          <div>
            <InvolvedTeamPeriod
              episodeOfCare={
                Object {
                  "careManager": Object {
                    "display": "Joe Bloggs (0113 000000)",
                    "typeName": "Care Manager",
                  },
                  "id": "690ca927-68e0-c511-b2c7-898056926ze4|involvedteam",
                  "metadata": Object {
                    "dataSources": Array [
                      Object {
                        "display": "Mock",
                      },
                    ],
                    "isRedacted": true,
                    "requestedWhen": "",
                  },
                  "period": Object {
                    "start": Object {
                      "kind": "DATE",
                      "value": "2019-12-01T00:00:00+00:00",
                    },
                  },
                  "status": "ACTIVE",
                  "team": Array [
                    Object {
                      "display": "LYPFT Inbound Care Team",
                      "typeName": "Test Team",
                    },
                  ],
                  "type": Array [
                    Object {
                      "coding": Array [
                        Object {
                          "code": "109016",
                          "display": "Home Care",
                        },
                      ],
                    },
                  ],
                }
              }
            >
              <div
                className="emotion-2"
              >
                01-Dec-2019 to Present
              </div>
            </InvolvedTeamPeriod>
            <InvolvedTeamType
              episodeOfCare={
                Object {
                  "careManager": Object {
                    "display": "Joe Bloggs (0113 000000)",
                    "typeName": "Care Manager",
                  },
                  "id": "690ca927-68e0-c511-b2c7-898056926ze4|involvedteam",
                  "metadata": Object {
                    "dataSources": Array [
                      Object {
                        "display": "Mock",
                      },
                    ],
                    "isRedacted": true,
                    "requestedWhen": "",
                  },
                  "period": Object {
                    "start": Object {
                      "kind": "DATE",
                      "value": "2019-12-01T00:00:00+00:00",
                    },
                  },
                  "status": "ACTIVE",
                  "team": Array [
                    Object {
                      "display": "LYPFT Inbound Care Team",
                      "typeName": "Test Team",
                    },
                  ],
                  "type": Array [
                    Object {
                      "coding": Array [
                        Object {
                          "code": "109016",
                          "display": "Home Care",
                        },
                      ],
                    },
                  ],
                }
              }
            >
              <div
                className="emotion-3"
              >
                Active
              </div>
            </InvolvedTeamType>
          </div>
        </div>
      </InvolvedTeamSummaryItem>
    </li>
  </ListItem>
</InvolvedTeamSummary>
`;
